README.md

```diff
@@ -7,9 +7,18 @@ This package provides the built-in `util.promisify` in node v8.0.0 and later, an
 
 ## Usage
 
+**Direct**
 ```js
 const promisify = require('util.promisify')
-// Use `promisify` just like the built-in
+// Use `promisify` just like the built-in method on `util`
+```
+
+**Shim**
+```js
+require('util.promisify').shim()
```

Comment:
It wasn't a mistake; it's there for convenience - but it's still nicer to encourage people to use deep exports :-)